metron-deployment/packaging/docker/rpm-docker/pom.xml

```diff
@@ -227,4 +183,97 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>InDocker</id>
+            <activation>
+                <file>
+                    <!-- If this file exists, then we are running in docker already -->
+                    <exists>/.dockerenv</exists>
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>1.5.0</version>
+                        <executions>
+                            <execution>
+                                <id>rpm-build</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>/bin/bash</executable>
+                                    <arguments>
+                                        <argument>-c</argument>
+                                        <argument>./build.sh ${project.version}</argument>
+                                    </arguments>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>NeedsDocker</id>
+            <activation>
```

Comment:
To clarify the way it works:

At metron-deploymnet:pom.xml => Should we build RPM's?
At rpm-docker:pom.xml => do we need to start docker or just call script